mlrun/datastore/base.py

```diff
@@ -107,7 +114,7 @@ def _get_secret_or_env(self, key, default=None, prefix=None):
 
     def get_storage_options(self):
         """get fsspec storage options"""
-        return None
+        return self._sanitize_storage_options(None)
```

Comment:
I think you can just return `{}` here.

Reply:
I want to enforce the pattern - always return via _sanitized_(...). If tomorrow we decide return None instead of {}  for example

Reply:
Doesn't really seem worthwhile, but I guess it's not very important either.